pages.es/osx/aiac.md

```diff
@@ -0,0 +1,24 @@
+# aiac
+
+> Utiliza OpenAI para generar configuraciones IaC, utilidades, consultas y más.
+> Más información: <https://github.com/gofireflyio/aiac>.
+
+- Genera Terraform para una cuenta de almacenamiento Azure:
+
+`aiac get terraform {{para una cuenta de almacenamiento azure}}`
```

Comment:
This application works with Spanish specs or English-only?

Reply:
English only

Reply:
applied english queries